test/e2e_node/memory_qos_eviction_test.go

```diff
@@ -0,0 +1,225 @@
+/*
```

Comment:
I don't even want to merge this!  I just put it up for comparison right now as I debug the original PR.